protos/bento/protos/services.proto

```diff
@@ -41,7 +41,10 @@ message ApplySimulationReq {
   // Definition of the simulation to create or apply
   SimulationDef simulation = 1;
 }
-message ApplySimulationResp {}
+message ApplySimulationResp {
+  // Simulations def with all unspecified entities created
```

Comment:
`unspecified entities` **ids** created right? I can fix this in PR #28

Reply:
Yeah, I missed that word out. I think I also didn't assign system IDs, since I just run the systems, no need to identify them at all.

Reply:
Resolved in a8ad07d